tests/test_cli/test_cfg_subcmd.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+import shutil
+
+from test_cli import capture
+this_folder = os.path.dirname(__file__)
+
+def test_cfg_help(python_version):
+    """Test if Manim successfully adds configparsers when a subcommand is invoked."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg --help"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0, f"The cfg subcommand is not working as intended."
+
+def test_cfg_show(python_version):
+    """Test if the `manim cfg show` command works as intended."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg show"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0
+    assert f"{os.path.sep}tests{os.path.sep}".encode("utf-8") in out, err
+
+def test_cfg_export(python_version):
+    """Test if the `manim cfg export` command works as intended."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg export --dir temp"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0
+    assert os.path.exists(os.path.join(this_folder,"temp","manim.cfg"))
+    with open(os.path.join(this_folder,"temp","manim.cfg"),"r") as writtencfg:
+        assert "sound = True" in writtencfg.read(), err
+    shutil.rmtree(os.path.join(this_folder,"temp"))
+
+def test_cfg_write(python_version):
+    """Simulate using the command `manim cfg write`"""
+    cfgfilepath = os.path.join(this_folder, "manim.cfg")
+    command = f"cd {this_folder} && {python_version} -m manim cfg write --level cwd"
+
+    """As the number of config values that `manim cfg write` can modify increases, so
+    must the number of newlines and/or values written in write_cfg_sbcmd_input increase."""
+    with open(cfgfilepath) as cfgfile:
+        original = cfgfile.read()
+
+    out, err, exitcode = capture(
+        command,
+        instream=open(os.path.join(this_folder, "write_cfg_sbcmd_input.txt")),
+        use_shell=True
+        )
+    assert exitcode == 0, err
+
+    with open(cfgfilepath,"r") as cfgfile:
+        assert "sound = False" in cfgfile.read()
+
+    with open(cfgfilepath,"w") as cfgfile:
```

Comment:
**Issue (blocking):** Ok, here, this is quite nitpicking. This can cause trouble when the test fail and nothing (or something wrong is written within the config file. As the test would have failed, this line will never be run and we will end up having a wrong config file for tests. 
You can fix that by using a pytest fixture : Here are two links that can help you : https://stackoverflow.com/questions/56997702/how-can-i-run-cleanup-code-only-for-specific-tests-with-pytest
https://docs.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects

(Don't hesitate to ask if we need help (although I'm not an expert on that))

Reply:
Thank you for providing those resources. I've written a `reset_cfg_file` fixture that is used in the test now.